### PR TITLE
added rotate command fixes #281

### LINKED ISSUE #281: Add 'granted credentials rotate' command
Add a `granted credentials rotate` command which generates new keys for the profile in AWS, and updates the profile.
It should work similarly to `granted credentials update`, but handle the generation of new access keys automatically.

_Originally posted by @ipmb in https://github.com/common-fate/granted/issues/178#issuecomment-1276180479_

### PR DESCRIPTION
**Pull request by n3s7or**
fixes #281

### CODE REVIEW

File: pkg/granted/credentials.go [hunk 1]
```diff
@@ -503,3 +508,76 @@ var ExportCredentialsCommand = cli.Command{
 		return nil
 	},
 }
+
+var RotateCredentialsCommand = cli.Command{
+	Name:      "rotate",
+	Usage:     "Generates new access key for the profile in AWS, and updates the profile",
+	ArgsUsage: "[<profile>]",
+	Action: func(c *cli.Context) error {
+		profileName := c.Args().First()
```
Comment (Eddie023) [inline pkg/granted/credentials.go:517]: You can make `profile` as optional flag like in your TokenExpiryCommand PR which will give a nice Usage description to users. Currently, it's not intuitive if we can bypass the profile selection if profile name is passed as first argument.

File: pkg/granted/credentials.go [hunk 2]
```diff
@@ -503,3 +508,76 @@ var ExportCredentialsCommand = cli.Command{
...
+		if err != nil {
+			return err
+		}	
+
+
+		fmt.Printf("Updated %s in secure storage\n", profileName)
+
+		return nil		
+	},
+}
```
Comment (Eddie023) [inline pkg/granted/credentials.go:583]: new line at EOF

File: pkg/granted/credentials.go [hunk 3]
```diff
@@ -503,3 +508,76 @@ var ExportCredentialsCommand = cli.Command{
...
+			return err
+		}
+
+		_, err = iamClient.UpdateAccessKey(c.Context, &iam.UpdateAccessKeyInput{AccessKeyId: &t.AccessKeyID, Status: "Inactive"})
+		if err != nil {
+			return err
+		}	
+
+
+		fmt.Printf("Updated %s in secure storage\n", profileName)
```
Comment (Eddie023) [inline pkg/granted/credentials.go:579]:
```suggestion
		cli.Success("Access Key of '%s' profile has been successfully rotated and updated in secure storage\n", profileName)
```

**Comment by n3s7or** [comment]
Thanks for the review @Eddie023 , I made a few changes according to your suggestions.

**Review by JoshuaWilkes — APPROVED** [review]

**Pull request merged**